pkg/sql/sem/tree/schema_helpers_test.go

```diff
@@ -58,13 +58,31 @@ func TestIsAllowedLDRSchemaChange(t *testing.T) {
 			stmt:      "ALTER TABLE t ADD COLUMN a INT, DROP COLUMN b",
 			isAllowed: false,
 		},
+		{
+			stmt:      "ALTER TABLE t ADD COLUMN a INT, SET (ttl = 'on', ttl_expiration_expression = 'expires_at')",
+			isAllowed: false,
+		},
+		{
+			stmt:      "ALTER TABLE t SET (ttl = 'on', ttl_expire_after = '5m')",
+			isAllowed: false,
+		},
+		{
+			stmt:      "ALTER TABLE t SET (ttl = 'on', ttl_expiration_expression = 'expires_at')",
```

Comment:
forgive my ignorance: why is the `ttl` param allowed for ALTER TABLE SET but not for ALTER TABLE RESET? Perhaps a comment in the source code could help here.

Reply:
i already had this comment in `IsAllowedLDRSchemaChange`. let me know if you have suggestions on what else to include in the comment.
```
				} else if slices.Contains(c.Params, "ttl") {
					// Resetting `ttl` can also result in the expiration column being
					// dropped.
					onlySafeStorageParams = false
				}
```